docs/modules/clusters/pages/ucn-member-side.adoc

```diff
@@ -0,0 +1,89 @@
+= Member Configuration
+:description: {ucn} are referenced by name, and can be statically configured in the Hazelcast configuration file. You can specify JAR and JARs In ZIP resources in the configuration file.
+:page-enterprise: true
+:page-beta: true
+
+{description}
+
+To configure {ucn} using the _hazelcast_ configuration file, in either XML or YAML format, complete the following steps:
+
+. If not already open, open the _hazelcast_ configuration file in your preferred editor and scroll to the {ucn} section
+. If required, add a `java-serialization-filter` to verify classes when loaded into the class loader during namespace deployment
++
+For further information on the configuration of the `java-serialization-filter`, see the xref:serialization:serialization-configuration.adoc[] topic.
```

Comment:
We should include some sort of `java-serialization-filter` example somewhere imo, as it's an important security feature and our customers value security